src/java/picard/analysis/CollectRrbsMetrics.java

```diff
@@ -64,41 +64,46 @@
-            "rates. The tool also outputs the numbers of reads having no CpG sites, and the numbers of reads discarded from the " +
-            "analysis due to inadequate size or excessive numbers of mismatches." +
-            "<br /><br />" +
-            "The tool also provides a table containing detailed information on CpG occurrence frequency, CpG conversion frequencies " +
-            "[C -> T (+ strand) or G -> A (- strand)], and the specific locations of the CpG sites in the genome. The conversion " +
-            "frequency helps determines the methylation status of a CpG site." +
-            "<br /><br />" +
-            "Finally, the tool provides graphical representation of four metrics in the form of a \".pdf\" document. These metrics " +
-            "are the bisulfite conversion rate for CpG and non-CpG cytosines, a distribution of the numbers of CpG sites as a " +
-            "function of CpG conversion rate, the distribution of CpG sites by read coverage, and the numbers of reads discarded due " +
-            "to high numbers of mismatches or inadequate read size." +
-            "" +
-            "<h4>Usage example:</h4>" +
-            "<pre>" +
-            "java -jar picard.jar CollectRrbsMetrics \\<br />" +
-            "      I=input.bam \\<br />" +
-            "      M=rrbs_metrics \\<br />" +
-            "      R=reference_sequence.fasta" +
-            "</pre>" +
-            "<hr />" +
-            "" +
-            "Please see " +
-            "<a href='https://broadinstitute.github.io/picard/picard-metric-definitions.html#RrbsCpgDetailMetrics'>" +
-            "the RrbsCpgDetailMetrics documentation</a> and the " +
-            "<a href='https://broadinstitute.github.io/picard/picard-metric-definitions.html#RrbsSummaryMetrics'>" +
-            "the RrbsSummaryMetrics documentation</a>for detailed explanations of the output metrics." +
-            "<hr />";
+    static final String USAGE_DETAILS = "<p>This tool calculates and reports QC metrics for reduced representation bisulfite " +
+            "sequencing (RRBS) data and is based on the methylation status of cytosine (C) bases in both CpG and non-CpG sites across all " +
+            "reads of a BAM/SAM file.  For detailed information about RRBS, please see " +
+            "<a href=\"http://gatkforums.broadinstitute.org/gatk/discussion/6330/bisulfite-sequencing-cytosine-methylation?new=1\"><strong>RRBS.</strong></a></p>"+
+    "<p>This tool outputs metrics in both tabular and graphical forms including the bisulfite conversion rate for CpG and non-CpG cytosines," +
+            " a distribution of the numbers of CpG sites as a function of CpG conversion rate, the distribution of CpG sites by read coverage," +
+            " and the numbers of reads discarded due to high numbers of mismatches or inadequate read size.</p>"+
+    "<center><h3>Arguments</h3></center>"+
```

Comment:
@dekling Why did you add a table of arguments? These already get added automatically when the docs get built. If you want to add details about specific arguments (within reason) you can add them in the docstrings that describe the options, just below this in the code.